bitrise.yml

```diff
@@ -1463,7 +1463,103 @@ workflows:
             App|${BITRISE_APP_URL}
             #mobile-testeng|https://mozilla.slack.com/archives/C02KDDS9QM9
             Mana|https://mana.mozilla.org/wiki/display/MTE/Mobile+Test+Engineering
-
+  Firebase_Performance_Test:
+    before_run:
+    - 1_git_clone_and_post_clone
+    - 2_certificate_and_profile
+    - 3_provisioning_and_npm_installation
+    steps:
+    - xcode-archive@4.7.2:
+        inputs:
+        - compile_bitcode: 'no'
+        - upload_bitcode: 'no'
+        - team_id: 9G8J6YA743
+        - export_method: development
+        - distribution_method: development
+        - export_development_team: 9G8J6YA743
+        - output_tool: xcodebuild
+        - scheme: Fennec_Enterprise
+        - configuration: Fennec_Enterprise
+        - project_path: "$BITRISE_PROJECT_PATH"
+    - deploy-to-bitrise-io@2: {}
+    - script@1.1:
+        title: Firebase Build for Testing Physical Devices
```

Comment:
This should work. In case you want to use predefined Bitrise steps, not sure if you have seen this documentation already:
https://devcenter.bitrise.io/en/testing/device-testing-with-firebase/device-testing-for-ios.html

Reply:
Using the default configuration may be easier, or maybe not 😅

Reply:
i've got this bookmarked and am looking at it for a substitute. might be a good follow up if my script is brittle or harder to work with than their default step. happy to use whichever.